suite-native/device/src/components/TrezorAnimation.tsx

```diff
@@ -10,11 +10,11 @@ const ANIMATION_WIDTH = 2000;
 const ANIMATION_HEIGHT = 2667;
 
 const animationStyle = prepareNativeStyle(({ borders }) => ({
-    position: 'absolute',
+    flexShrink: 1,
+    width: '100%',
     aspectRatio: ANIMATION_WIDTH / ANIMATION_HEIGHT,
+    alignSelf: 'center',
```

Comment:
Nit: Would move this right above/below `flexShrink` as they seem related. Feel free to ignore.